diff --git a/config/boot.rb b/config/boot.rb
index 0ad0f78..2b51e76 100644
--- a/config/boot.rb
+++ b/config/boot.rb
@@ -1,7 +1,7 @@
 # Don't change this file!
 # Configure your app in config/environment.rb and config/environments/*.rb
 
-RAILS_ROOT = "#{File.dirname(__FILE__)}/.." unless defined?(RAILS_ROOT)
+RAILS_ROOT = File.expand_path("#{File.dirname(__FILE__)}/..") unless defined?(RAILS_ROOT)
 
 module Rails
   class << self
diff --git a/script/server b/script/server
index 3c67f39..94b4b9c 100755
--- a/script/server
+++ b/script/server
@@ -1,3 +1,3 @@
 #!/usr/bin/env ruby
-require File.dirname(__FILE__) + '/../config/boot'
+require File.expand_path(File.dirname(__FILE__) + '/../config/boot')
 require 'commands/server'
