diff --git a/config/boot.rb b/config/boot.rb
index 98ff2e0..8ddeb3f 100644
--- a/config/boot.rb
+++ b/config/boot.rb
@@ -23,7 +23,7 @@ def find_home
 end
 
 unless defined?(RAILS_ROOT)
-  root_path = File.join(File.dirname(__FILE__), '..')
+  root_path = File.expand_path(File.join(File.dirname(__FILE__), '..'))
 
   unless RUBY_PLATFORM =~ /mswin32/
     require 'pathname'
@@ -73,4 +73,4 @@ unless defined?(Rails::Initializer)
   end
 
   Rails::Initializer.run(:set_load_path)
-end
\ No newline at end of file
+end
