diff --git a/app/models/build.rb b/app/models/build.rb
index da49a54..67a2f7b 100644
--- a/app/models/build.rb
+++ b/app/models/build.rb
@@ -167,7 +167,7 @@ EOF
     # --nosearch flag here prevents CC.rb from building itself when a project has no Rakefile
     # ARGV.clear at the end prevents Test::Unit's AutoRunner from doing anything silly, like trying to require 'cc:rb'
     # Some people saw it happening.
-    %{#{Platform.interpreter} -e "require 'rubygems' rescue nil; require 'rake'; load '#{ABSOLUTE_RAILS_ROOT}/tasks/cc_build.rake'; ARGV << '--nosearch'#{CruiseControl::Log.verbose? ? " << '--trace'" : ""} << 'cc:build'; Rake.application.run; ARGV.clear"}
+    %{#{Platform.interpreter} -e "require 'rubygems' rescue nil; require 'rake'; load '#{ABSOLUTE_RAILS_ROOT}/tasks/cc_build.rake'; ARGV << '--nosearch'#{CruiseControl::Log.verbose? ? " << '--trace'" : ""} << 'cc:build'; Rake.application.run; ARGV.clear; Test::Unit.run=true"}
   end
 
   def in_clean_environment_on_local_copy(&block)
