diff --git a/lib/source_control/git/log_parser.rb b/lib/source_control/git/log_parser.rb
index a6318a0..158c5a9 100644
--- a/lib/source_control/git/log_parser.rb
+++ b/lib/source_control/git/log_parser.rb
@@ -12,7 +12,7 @@ module SourceControl
           case line
           when /^commit /
             revisions << revision = Revision.new
-            revision.number = line.split[1][0..6]
+            revision.number = line.split[1][0..9]
             
           when /^author /
             revision.author, revision.time = read_author_and_time(line)
